src/modules/naomi/naomiAntiphish.ts

```diff
@@ -18,7 +18,19 @@ export const naomiAntiphish = async (Becca: BeccaLyria, message: Message) => {
     const result = await axios.post<{ message: string }>(
       "https://bad-domains.walshy.dev/report",
       { domain: link }
-    );
+    ) &&
+      axios.post<boolean>(
+      `http://heptagrambotproject.com/api/v0/api/scam/link/post`,
+      {
+      headers: {
+        Authorization: "Bearer " + config.heptagramApiToken,
+      },
+      body: {
+        link: link,
+        reportedby: `${message.author.id} || Becca Lyria`,
+      },
+    });
+    ;
```

Comment:
```suggestion
    const heptagramResult = await axios.post<boolean>(
      `http://heptagrambotproject.com/api/v0/api/scam/link/post`,
      {
        headers: {
          Authorization: "Bearer " + Becca.configs.heptagramApiToken,
        },
        body: {
          link: link,
          reportedby: `${message.author.id} || Becca Lyria`,
        },
      }
    );
```
Separate these into two calls.